frontend/src/components/chat/acp/__tests__/state.test.ts

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2025 Marimo. All rights reserved. */
+/* Copyright 2024 Marimo. All rights reserved. */
```

Comment:
For follow up PR, but we should have a check that harmonizes the year for these copyright statements